packages/react-components/src/shared/src/slots.tsx

```diff
@@ -1,19 +1,31 @@
-import { Children, useMemo } from "react";
-import { augmentElement, resolveChildren } from "../../shared";
+import { augmentElement, resolveChildren } from "..";
 import { isNil, isString, isUndefined } from "lodash";
+import React, { Children, ComponentType, ReactElement, ReactNode, useMemo } from "react";
```

Comment:
I don't think you need to import the React object.

Reply:
<Wrapper>{children}</Wrapper>

this is jsx in this file